Routes.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express")
+const app = express()
+const port =  3001
+
+const villanInfo = [
```

Comment:
The data (villanInfo) and the routes are all in the same file. Consider separating them into different modules for better modularity and readability. [medium]